hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -368,7 +387,7 @@ instance Arbitrary (TxIdType tx) => Arbitrary (RequirementFailure tx) where
 data Outcome tx
   = NoOutcome
   | Effects {effects :: [Effect tx]}
-  | NewState {headState :: HeadState tx}
+  | NewState {events :: [HeadStateEvent tx]}
```

Comment:
This type should only carry one event. If we really need several events, we can still `Combined` stuff.

We should name it differently also.